src/App.tsx

```diff
@@ -1,61 +1,125 @@
+import React, { useState } from 'react';
 import './App.scss';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+import { TodoList } from './components/TodoList';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+function getUserById(userId: number) {
+  return usersFromServer.find(user => user.id === userId)
+    || null;
+}
 
 export const App = () => {
+  const [titleValue, setTitleValue] = useState('');
+  const [selectValue, setSelectValue] = useState(0);
+  const [showTitleError, setTitleError] = useState(false);
+  const [showSelectError, setSelectError] = useState(false);
+
+  const [todos, setTodos] = useState(todosFromServer.map(todo => ({
+    ...todo,
+    user: getUserById(todo.userId),
+  })));
+
+  const getmaxTodoId = () => {
+    return Math.max(...todos.map(todo => todo.id), 0);
+  };
+
+  const hendleTitleChange = (event: React.ChangeEvent<HTMLInputElement>) => {
```

Comment:
Typo in the function name `hendleTitleChange`. It should be `handleTitleChange`.